build_deps/tf_dependency/BUILD.tpl

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "tf_header_lib",
+    hdrs = [":tf_header_include"],
```

Comment:
the header file inclusion and libtensorflow_framework inclusion seems correct to me.
Though it'd be better to point out where we're relying this from. The main reason for this ask is that "tensorflow_framework" might not be a guaranteed binary in the long run, and we should know how to fix this once that becomes true.

Reply:
I'm not sure I follow -- do you mean comments specifying that this is enabling linking to the installed TF version, or something else?